src/arbeidssøkerskjema/steg/2-Oppsummering.tsx

```diff
@@ -56,6 +57,10 @@ const Oppsummering: React.FC = () => {
   );
   const spørsmålOgSvar = VisLabelOgSvar(skjema.arbeidssøker);
 
+  useEffect(() => {
```

Comment:
Dette er bara såpirk, men hade det gått att lage useOnMount hook ala som tar in en callback som argument
ala
const useOnMount = (effectCallback) = > {
   useEffect(effectCallBack, [])
}

Reply:
då blir resultatet useOnMount(logSidevisningArbeidssokerskjema('Oppsummering'))
og useOnMount(logSidevisningArbeidssokerskjema('Kvittering'));

osv

Reply:
God idé, gjorde det i 741b683